### ISSUE TITLE
Standardize single quotation marks in checker error messages

### ISSUE THREAD

**Issue by gligneul** [pallene-lang/pallene#135]
Some messages use quotation marks to enclose names (functions, variables, or types); for instance:
```
    it("forbids variables from being used before they are defined", function()
        assert_error([[
            function fn(): nil
                x = 17
                local x = 18
            end
        ]],
            "variable 'x' is not declared")
    end)
```
However, in other cases there is no quotation marks, for instance:
```
    it("catches assignment to function", function ()
        assert_error([[
            function f()
            end

            function g()
                f = g
            end
        ]],
            "attempting to assign to toplevel constant function f")
    end)
```
We should decide what is the best and stick with it.

**Comment by hugomg** [comment]
I vote on using single quotes, like `'f'`

1. Lua uses single quotes. the error message for calling a non-existing function is `attempt to call a nil value (global 'f')`
2. Clang seems to use single quotes too: `warning: implicit declaration of function 'f' is invalid in C99`
3. GCC uses fancy single quotes: `warning: implicit declaration of function ‘f’`

**Comment by ahmedkrmn** [comment]
Will start working on this and submit a PR once finished.

**Comment by hugomg** [comment]
Fixed by #142
